README.md

```diff
@@ -2,65 +2,23 @@
+This project is following *Herodotos* documentation organization, please use the [**documentation entrypoint**](docs/README.md) to find out more information about this project.
 
-- [MathType for Atto](https://github.com/wiris/mooodle-atto_wiris)
-- [MathType for TinyMCE 3.x](https://github.com/wiris/moodle-tinymce_tiny_mce_wiris) for Moodle versions under 4.2.
-- [MathType for TinyMCE 6.x](https://github.com/wiris/moodle-tinymce6_wiris) for Moodle versions above 4.2.
+## Who's to Blame
 
-## Installation
+The [issue reporting](docs/ISSUE_REPORTING.md) document provides insights on how to report an issue on github or the support team.
 
-Install the plugin like any other plugin to folder `/filter/wiris`.
+## Changelog
 
-You can use git:
+Since version 7.24.0, all notable changes to this project are documented in the [Changelog](./CHANGELOG.md) file.
 
-```sh
-$ git clone https://github.com/wiris/moodle-filter_wiris.git filter/wiris
-```
-
-Alternatively, you can [download the plugin](https://github.com/wiris/moodle-filter_wiris/archive/main.zip) and unzip the file into filters folder, and then rename the new folder to `wiris`.
-
-## Uninstall
-
-This project is a dependency of the following:
-
-* MathType for Atto
-* MathType for TinyMCE
-* MathTypw for WirisQuizzes
-
-
-To uninstall the Wiris filter in Moodle, all the previous plugins mentioned must also be uninstalled. If any of these plugins are still installed, the Wiris filter cannot be removed.
-
-
-## Releases
-
-Since version 7.24.0, all notable changes to this project are documented in the [CHANGES.md](CHANGES.md) file. You can download any release of this plugin from the [Official Moodle's page](https://moodle.org/plugins/filter_wiris).
-
-## Libraries
-
-The MathType Web Integration JavaScript SDK is released as a npm package: [@wiris/mathtype-html-integration-devkit](https://www.npmjs.com/package/@wiris/mathtype-html-integration-devkit).
-
-This plugin uses the **MathType Web Integration JavaScript SDK** ([@wiris/html-integrations](https://github.com/wiris/html-integrations)), released under GNU GPLv3 license as a npm package: [@wiris/mathtype-html-integration-devkit](https://www.npmjs.com/package/@wiris/mathtype-html-integration-devkit).
-
-> The library's source code can be found at [@wiris/html-integrations](https://www.github.com/wiris/html-integrations) repository
-
-**Note:** More details on the `thirdpartylibs.xml` file.
-
-## Contributing
-
-We would love for you to contribute to this project and help make it better.
-
-As a contributor, the guidelines we would like you to follow are documented in the [CONTRIBUTING.md](CONTRIBUTING.md) file.
-
-## Technical Support
-
-If you have questions or need help integrating MathType, please contact us (support@wiris.com) instead of opening an issue.
+You can download any release of this plugin from the [Official Moodle's page](https://moodle.org/plugins/filter_wiris).
```

Comment:
Me temo que esto no cumple con la plantilla de Herodotos... 😢

Reply:
¡Cierto! Esto lo tomé como referencia del add-in y también, al ser un repositorio público, para que haya un punto de entrada con información básica. ¿Crees que debería seguir estrictamente la plantilla?

Reply:
Si, dentro de lo posible si! Es lo más fácil, y si hubiera algo que queremos poner pero no está contemplado en la plantilla, lo hablamos!

Reply:
Perfecto!

Reply:
Done!